cve_bin_tool/output_engine/html.py

```diff
@@ -92,41 +98,30 @@ def output_html(
                     fix_id=hid,
                 )
 
-                # generate analysis_data
-                if cve.severity.lower() == "critical":
-                    analysis_data["critical"] += 1
-                elif cve.severity.lower() == "high":
-                    analysis_data["high"] += 1
-                elif cve.severity.lower() == "medium":
-                    analysis_data["medium"] += 1
-                elif cve.severity.lower() == "low":
-                    analysis_data["low"] += 1
-                else:
-                    pass
+            analysis_data = Counter([cve.severity for cve in product.cves])
 
             # initialize a figure object for Analysis Chart
+            labels = []
+            values = []
+            for key, value in analysis_data.items():
+                if value > 0:
+                    labels.append(key)
+                    values.append(value)
```

Comment:
You don't need to check this because in our case key in Counter will only get created if it has nonzero value.
So, simply following code will be enough if you need keys and values separately.
```suggest
labels = analysis_data.keys()
values = analysis_data.values()
```